### DISCUSSION TITLE
deprecated bundledWebRuntime

### DISCUSSION THREAD

**Discussion by MrAmericanMike** [ionic-team/capacitor#7436] [Q&A] [answered]
I just used `pnpm create @capacitor/app` 

Then when doing `pnpm cap add android` I get this warning:

Warning:
```
[warn] The bundledWebRuntime configuration option has been deprecated. Can be safely deleted.
```

Been that this sets up a project with "latest" versions of Capacitor, shouldn't the `capacitor.config.json` already remove that deprecated option?

**Comment by jcesarmobile** [comment, marked as answer]
Thanks, I've just released a new version of `@capacitor/create-app` that removes the `bundledWebRuntime` entry from the `capacitor.config.json.`
